src/lifetimes.md

```diff
@@ -47,21 +48,24 @@ likely desugar to the following:
 
 <!-- ignore: desugared code -->
 ```rust,ignore
-// NOTE: `'a: {` and `&'b x` is not valid syntax!
-'a: {
+// NOTE: `'global_lifetime: {` and `&'x_lifetime x` is not valid syntax!
+'global_lifetime: {
     let x: i32 = 0;
-    'b: {
-        // lifetime used is 'b because that's good enough.
-        let y: &'b i32 = &'b x;
-        'c: {
-            // ditto on 'c
-            let z: &'c &'b i32 = &'c y;
+    'x_lifetime: {
+        // lifetime used is 'x_lifetime because that's good enough.
+        let y: &'x_lifetime i32 = &'x_lifetime x;
+        'y_lifetime: {
+            // ditto on 'y_lifetime
+            let z: &'y_lifetime &'x_lifetime i32 = &'y_lifetime y;
```

Comment:
After some re-thinking of this, I still think this is too descriptive. We easily can find `'a` is a lifetime and curly braces are scopes if we're familiar with Rust's syntax. That is, _naming_ lifetimes looks redundant, and tweaking prior wording should be enough here.